lib/plsql/connection.rb

```diff
@@ -197,9 +197,8 @@ def session_id
       @session_id ||= select_first("SELECT TO_NUMBER(USERENV('SESSIONID')) FROM dual")[0]
     end
 
-    # Set time zone (default taken from TZ environment variable)
+    # Set time zone
     def set_time_zone(time_zone=nil)
-      time_zone ||= ENV['TZ']
```

Comment:
With such a change a call on [spec/plsql/connection_spec.rb#L10](https://github.com/rsim/ruby-plsql/blob/master/spec/plsql/connection_spec.rb#L10) becomes redundant. Maybe we can get rid of it?

I wonder about the default value: should we keep it `nil` or make it `local` instead?